gui/additionsPane.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, parent):
         projectedImg = BitmapLoader.getImage("projected_small", "gui")
         gangImg = BitmapLoader.getImage("fleet_fc_small", "gui")
         cargoImg = BitmapLoader.getImage("cargo_small", "gui")
+        notesImg = BitmapLoader.getImage("skill_small", "gui")
```

Comment:
Bummer. Not seeing anything.
 
I recognize some of the icons in gui, I just can't remember the name of the set to search and grab an editor looking icon.